controls/busyindicator/integration.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Integration
```

Comment:
I am not entirely convinced this one should be a page on its own rather than a how-to KB. What's the case for having a separate page?

Furthermore, the Integration title in the navigation might mislead users that this is going to be a more general document, rather than a very specific integration. If you decide to keep this as a separate article (rather than a KB), change the title to `CollectionView Integration`.